todo_app/tests/test_utils.py

```diff
@@ -0,0 +1,34 @@
+import json
+import os
+
+from dotenv import find_dotenv
+
+
+class StubResponse():
+    def __init__(self, fake_response_data, status_code=200):
+        self.fake_response_data = fake_response_data
+        self._status_code = status_code
+
+    def json(self):
+        return self.fake_response_data
+
+    @property
+    def status_code(self):
+        return self._status_code
+
+
+def stub(url, params={}):
+    test_board_id = os.environ.get('TRELLO_BOARD_ID')
+    if url == f'https://api.trello.com/1/boards/{test_board_id}/cards':
+        return mock_get_cards_endpoint()
+    elif url == 'https://api.trello.com/1/boards':
+        return StubResponse([{'id': test_board_id}])
+    raise Exception(f'Integration test did not expect URL "{url}"')
+
+
+def mock_get_cards_endpoint():
+    mock_data_file_path = find_dotenv('trello_cards_mock_data.json')
```

Comment:
This feels a little odd - we're using a `find_dotenv` method to find something that definitely isn't a dotenv file! This fundamentally works but feels like behaviour the package owner could later change without warning (e.g. throw an error if the file found isn't in the right format) so isn't a great one to rely on

Not obvious but the path we want will be relative to the root folder, so we could set it directly:
`mock_data_file_path = 'todo_app/tests/trello_cards_mock_data.json'`